packages/twenty-front/src/utils/currentPage.ts

```diff
@@ -0,0 +1,11 @@
+import { useLocation } from 'react-router-dom';
+
+/**
+ * Custom hook to check if the current route matches the given path.
+ * @param path The route to match.
+ * @returns True if the current route matches the given path, false otherwise.
+ */
+export const useIsRoute = (path: string): boolean => {

```

Comment:
a few feedbacks about this file:
1) we don't add JS doc in the project as we try to avoid comments and we are already using typescript to provide a nice dev xp. Function names should be clear enough to avoid doing it
2) useIsRoute is a hook so it should not be in a utils folder
3) I don't thin this s actually useful, we can just do useLocation().pathname === path whenever we need

Reply:
So I would remove this hook actually

Reply:
Right. I'll remove it.

I was also thinking the same